pkg/logs/client/http/destination_test.go

```diff
@@ -366,3 +367,129 @@ func TestDestinationHA(t *testing.T) {
 		assert.Equal(t, isEndpointMRF, isDestMRF)
 	}
 }
+
+func TestTransportProtocol_HTTP1(t *testing.T) {
+	c := configmock.New(t)
+	assert.True(t, c.IsKnown("logs_config.transport_type"), "Config key logs_config.transport_type should be known")
+
+	// Force client to use HTTP/1
+	c.SetWithoutSource("logs_config.transport_type", "http1")
+	// Skip SSL validation
+	c.SetWithoutSource("skip_ssl_validation", true)
+
+	s := NewTestHTTPSServer(false)
+	defer s.Close()
+
+	timeout := 5 * time.Second
+	// Force HTTP/1 transport
+	client := httpClientFactory(timeout, c)()
+
+	// Create an HTTP/1.1 request
+	req, err := http.NewRequest("POST", s.URL, nil)
+	if err != nil {
+		t.Fatalf("Failed to create request: %v", err)
+	}
+
+	// Client send an HTTP1 request
+	resp, err := client.Do(req)
+	if err != nil {
+		t.Fatalf("Failed to make request: %v", err)
+	}
+	defer resp.Body.Close()
+
+	// Assert the protocol is HTTP/1.1
+	assert.Equal(t, "HTTP/1.1", resp.Proto)
```

Comment:
Shouldn't this be `HTTP/1` rather than `HTTP/1.1`?  Maybe a test/option is needed to accepts `http1_1`.